packages/tasit-account/lib/tasitAccount.js

```diff
@@ -0,0 +1,29 @@
+"use strict";
```

Comment:
@marcelomorgado Sometimes I see this type of dir being named `dist` rather than `lib`. How would you feel about that renaming?

Reply:
Ok, I'll do that.